deployer/cli.py

```diff
@@ -527,16 +517,37 @@ def create_pipeline(
             raise e
 
         console.print(
-            f"Pipeline '{pipeline_name}' created at '{pipeline_filepath}'"
-            f" with config files: {[str(p) for p in config_dirpath.glob('*')]}. :sparkles:",
+            f"\n Pipeline '{pipeline_name}' created at '{pipeline_filepath}'"
+            f"\n with config files: {[str(p) for p in config_dirpath.glob('*')]}. :sparkles: \n",
             style="blue",
         )
 
 
 @app.command(name="init")
 def init_deployer(ctx: typer.Context):
-    console.print("Welcome to Vertex Deployer!", style="blue")
-    console.print("This command will help you getting fired up.", style="blue")
+    def create_initial_pipeline(ctx: typer.Context, deployer_settings: DeployerSettings):
```

Comment:
So i think you don't need to have a function and can directly use the code inside `create_initial_pipeline` in `init_deployer` fore better readability